Assets/MRTK/Examples/Common/Profiles/RiggedHands/RiggedHandConfigurationProfile.asset

```diff
@@ -15,7 +15,7 @@ MonoBehaviour:
   isCustomProfile: 1
   targetExperienceScale: 3
   enableCameraSystem: 1
-  cameraProfile: {fileID: 11400000, guid: 1e70959308adb3541b9a29bb6c707e52, type: 2}
+  cameraProfile: {fileID: 11400000, guid: 8089ccfdd4494cd38f676f9fc1f46a04, type: 2}
```

Comment:
The previously referenced camera profile doesn't appear to exist in our repo. Replaced with our default profile